packages/core/hooks/precompact-dump.py

```
@@ -0,0 +1,199 @@
+
+try:
+    from event_logger import emit_event as _emit_event  # type: ignore[import-not-found]
+except ImportError:  # pragma: no cover - audit 未導入時のフォールバック
+    _emit_event = None  # type: ignore[assignment]
+
+# 保存するダンプファイルの最大数（古いものから削除）
+MAX_DUMP_FILES = 20
+
+
+def _now_stamp() -> str:
+    """ファイル名用のタイムスタンプを返す（UTC, コロン無し）。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)